frontend/src/pages/Inbox/components/Email.js

```diff
@@ -80,7 +83,8 @@ function EmailAttachments({ props }) {
                   attachment,
                 },
               }}
-              onClick={() => {
+              onClick={async () => {
+                await getAllFiles();
```

Comment:
instead of this you should use the `loadFiles` action that lives in `fileManagerActions`
```
import { loadFiles } from '../../../actions/fileManagerActions';

await loadFiles();
```